rest/replicator_test.go

```diff
@@ -3933,6 +3932,23 @@ func TestSGR2TombstoneConflictHandling(t *testing.T) {
 	}
 
 	for _, test := range tombstoneTests {
+		// requireTombstone validates tombstoned revision.
+		requireTombstone := func(bucket *base.TestBucket, docID string) {
+			var rawBody db.Body
+			_, err := bucket.Get(docID, &rawBody)
+			if base.TestUseXattrs() {
+				require.Error(t, err)
+				require.Equal(t, gocb.ErrKeyNotFound, pkgerrors.Cause(err))
```

Comment:
Maybe a question for Adam, is (v1) `gocb.ErrKeyNotFound` compatible with GoCB v2?

Reply:
Sure, let's check with Adam and see whether this needs to be removed.